src/Alac/AlacTagger.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace IndieHD\AudioManipulator\Alac;
+
+use getID3;
+use getid3_writetags;
+
+use Psr\Log\LoggerInterface;
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+
+use IndieHD\AudioManipulator\Validation\ValidatorInterface;
+use IndieHD\FilenameSanitizer\FilenameSanitizerInterface;
+use IndieHD\AudioManipulator\Tagging\AudioTaggerException;
+use IndieHD\AudioManipulator\Processing\Process;
+use IndieHD\AudioManipulator\Processing\ProcessInterface;
+use IndieHD\AudioManipulator\Processing\ProcessFailedException;
+use IndieHD\AudioManipulator\Tagging\TaggerInterface;
+use IndieHD\AudioManipulator\CliCommand\AtomicParsleyCommandInterface;
+
+class AlacTagger implements TaggerInterface
+{
+    public function __construct(
+        getID3 $getid3,
+        getid3_writetags $writeTags,
+        ProcessInterface $process,
+        LoggerInterface $logger,
+        FilenameSanitizerInterface $filenameSanitizer,
+        AtomicParsleyCommandInterface $command,
+        ValidatorInterface $validator
+    ) {
+
+        $this->getid3 = $getid3;
+        $this->writeTags= $writeTags;
+        $this->process = $process;
+        $this->logger = $logger;
+        $this->filenameSanitizer = $filenameSanitizer;
+        $this->command = $command;
+        $this->validator = $validator;
+
+        // This option is specific to the tag READER (the WRITER has its own,
+        // separate encoding setting).
+
+        $this->getid3->setOption(['encoding' => 'UTF-8']);
+
+        // TODO Make the log location configurable.
+
+        $fileHandler = new StreamHandler(
```

Comment:
Shame on me. I'll fix this shortly.

Reply:
Actually, upon further consideration, the scenario here is the same as I [described elsewhere in this PR](https://github.com/indiehd/audio-manipulator/pull/9/files#r280066346), although, I didn't create the proxy class as I did in the aforementioned case.

Before I change anything here, let's work through the other discussion, as the same solution applies here.

Reply:
I know how I'm going to fix this, but it's beyond the scope of this PR. Let's table this for now and I'll fix it in a separate PR.

Reply:
Created #13 to track this.